cookbooks/bcpc/attributes/default.rb

```diff
@@ -106,9 +136,44 @@
     'http://s3.amazonaws.com/dev.hortonworks.com/HDP/ubuntu12/2.x/BUILDS/2.3.6.2-2'
 
   repos['hdp_utils'] =
-    'http://s3.amazonaws.com/dev.hortonworks.com/HDP-UTILS-1.1.0.20/repos/ubuntu12'
+    'http://public-repo-1.hortonworks.com/HDP-UTILS-1.1.0.20/repos/ubuntu12'
```

Comment:
The documented URLs are on the public-repo-1 CNAME.

http://docs.hortonworks.com/HDPDocuments/Ambari-2.1.0.0/bk_Installing_HDP_AMB/content/_hdp_stack_repositories.html